src/Stats/Persister/RedisPersister.php

```diff
@@ -23,6 +23,11 @@ final class RedisPersister implements PersisterInterface
     private string $keyList;
 
     /** @param Redis<string, Redis> $redis */
+    /*
+     * @todo done in this way, custom args are never used if we want to customize only one of those
+     * (passing null is not feaseable, passing en empty string would result in empty string)
+     * Maybe is better to use ?string as typehint?
+     */
```

Comment:
I don't know! Maybe i prefer `Redis $redis`.
WDYT @garak ?

Reply:
This comment refers to other arguments, not `$redis` one, that's already typehinted.

Reply:
ok now I understand, you refer to `keyPrefix`. Ok for `?string` or let's clean it up and remove it! ;)

Reply:
I've removed the keys as in container build, we never pass them.